ar_exercise/exercise.rb

```diff
@@ -34,38 +34,108 @@ class Apartment < ActiveRecord::Base
+damien_corners = Apartment.where(address: "6005 Damien Corners")
+
 # get all tenants in that apartment
+damien_corners.tenants
+
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+
+all_tenants.each do |tenant|
+  puts tenant.name
+  puts tenant.id
+end
+
 # Iterate over each apartment, for each apartment, display it's address and rent price
+all_apartments = Apartment.all
+all_apartments.each do |apartment|
+  puts apartment.address
+  puts appartment.monthly_rent
+  puts "*" * 50
+end
+
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+all_apartments.each do |apartment|
+  apartment.address
+  apartment.tenants.each do |tenant|
+    puts tenant.name
+  end
+end
 
 ################################################
 # CREATING / UPDATING / DELETING
 ################################################
 
-# Hint, the following methods will help: `new`, `create`, `save`, `uddate`, `destroy`
+# Hint, the following methods will help: `new`, `create`, `save`, `update`, `destroy`
 
 # Create 3 new apartments, and save them to the DB
+apartment1 = Apartment.create(address: "4128 Baldwin Crescent", monthly_rent: 7500, sqft: 2000, num_beds: 3, num_baths: 2.5)
+apartment2 = Apartment.create(address: "4055 Joshua Barney", monthly_rent: 5000, sqft: 1500, num_beds: 2, num_baths: 2)
+apartment3 = Apartment.create(address: "1211 Abraham Drive", monthly_rent: 10000, sqft: 3500, num_beds: 5, num_baths: 4)
+
+
 # Create at least 9 new tenants and save them to the DB. (Make sure they belong to an apartment)
+new_tenant1 = Tenant.create(name: "Alexander	Hardy", age: "25", gender: "Male", apartment_id: 7)
+new_tenant2 = Tenant.create(name: "Sonia Bowen", age: "42", gender: "Female", apartment_id: 14)
+new_tenant3 = Tenant.create(name: "Raymond	Adams", age: "28", gender: "Male", apartment_id: 5)
+new_tenant4 = Tenant.create(name: "Tyler	French", age: "34", gender: "Male", apartment_id: 12)
+new_tenant5 = Tenant.create(name: "Carrie	Bryant", age: "29", gender: "Female", apartment_id: 9)
+new_tenant6 = Tenant.create(name: "Allan	Stewart", age: "43", gender: "Male", apartment_id: 19)
+new_tenant7 = Tenant.create(name: "Lynne	Yates", age: "22", gender: "Female", apartment_id: 9)
+new_tenant8 = Tenant.create(name: "Kelly	Grant", age: "41", gender: "Female", apartment_id: 17)
+new_tenant9 = Tenant.create(name: "Ricardo	Benson", age: "36", gender: "Male", apartment_id: 10)
+new_tenant10 = Tenant.create(name: "Valerie	Conner", age: "27", gender: "Female", apartment_id: 11)
+
+
 # Note: you'll use this little bit of code as a `seeds.rb` file later on.
 
 # Birthday!
 # It's Kristin Wisoky's birthday. Find her in the DB and change her age to be 1 year older
 # Note: She's in the seed data, so she should be in your DB
+kristin_wisoky = Tenant.find_by(name: "Kristin Wisoky")
+puts kristin_wisoky.age
+kristin_wisoky.age = kristin_wisoky.age + 1
+puts kristin_wisoky.age
```

Comment:
Technically this will only update the in-memory copy of the object. We weren't specific, but in this case you'd probably want to call `.save` to save to the DB. Or you can use `.update` or `.increment`.

```
kristin.save
```